CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 28.0.0
+
+- Removed: `stylelint` less than `14.11.0` from peer dependencies.
+- Changed: `alpha-value-notation` to be `"number"` for SVG `*-opacity` properties.
```

Comment:
[note] Follows the entry below:

https://github.com/stylelint/stylelint-config-standard/blob/c12b8cb2b9a57e8dbd2696dd7ac2f78e72b829a4/CHANGELOG.md?plain=1#L23